Needinfo for bugs with inconsistent severity flags (#1313)
mozilla/relman-auto-nag: templates/severity_inconsistency_needinfo.txt

> The severity field for this bug is set for to {{ extra[bugid]["severity"] }}, however, Whiteboard shows a higher severity with the flag {{ extra[bugid]["whiteboard_severity"] }}.

Reviewer: ```suggestion
The severity field for this bug is set to {{ extra[bugid]["severity"] }}. However, the accessibility severity is higher, {{ extra[bugid]["whiteboard_severity"] }}.
```

> :{{ nickname }}, could you have a look please?

Reviewer: ```suggestion
:{{ nickname }}, could you consider increasing the severity?
```

> {{ documentation }}

Reviewer: CC @eeejay @jcsteh @MReschenberg, in case you have any preference for the wording of the needinfo

Reviewer 2: This looks good to me :)
